.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

```diff
@@ -0,0 +1,25 @@
+---
+name: 💡 Feature Request
+about: Report a Backend bug on Kubeflow Pipelines
+title: "[feature request]: [Description]"
+labels: kind/feature
+---
+
+### Feature Area
+
+<!-- Uncomment the labels below which are relevant to this feature: -->
+<!-- /area frontend -->
+<!-- /area backend -->
+<!-- /area sdk -->
+<!-- /area samples -->
+<!-- /area components -->
+
```

Comment:
I think an important missing section is `Use Case` or `Problem` (I'm not exactly sure about the wording here).

Sometimes, users may not have the best context to design a feature that solves their problem. Providing context for what problem to solve or the scenario helps us better understand the real problem.

Reply:
That makes sense, here is how I reword the questions:


### What is the use case or pain point?

--> It helps us understand the benefit of this feature for your use case.

### Is there a workaround currently?

--> Without this feature, how do you accomplish your task today?

Reply:
Nice! Love the new sections

Reply:
That is great! Thank you for the suggestion.